### ISSUE TITLE
Python 2 to 3 conversion should update the shebang

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#255]
So the newly converted code automatically runs on Python 3 and not 2. It could also be useful to add a Python 3 shebang if there wasn't one before.

**Comment by zrzka** [comment]
Not sure if Pythonista should interfere here. If it's not added by modernize / 2to3, I think it's perfectly ok to not add it. Power user can add it and he knows what he is doing, common user have no idea what the shebang line is. @omz WDYT?

**Comment by omz** [comment]
I agree with @dgelessus that it would make sense to add it, though it's probably not that important, as Py3 is the default anyway.
